pkg/api/validate/dynamic.go

```diff
@@ -43,13 +45,15 @@ type dynamic struct {
 	oc    *api.OpenShiftCluster
 	vnetr *azure.Resource
 
+	code string
+	typ  string
```

Comment:
Please raise follow-up to try to get this generalized so this would be set by called instead in the initiation